tests/test_binaries.py

```diff
@@ -120,6 +120,25 @@ def testArgumentsAssertions(self):
         expected = self._loadVisitedSet(os.path.join(dirname, 'reference', 'arguments_linux_amd64_visited.txt'))
         self.assertGreaterEqual(actual, expected)
 
+    def testArgumentsAssertionsArmv7(self):
+        dirname = os.path.dirname(__file__)
+        filename = os.path.abspath(os.path.join(dirname, 'binaries/arguments_linux_armv7'))
+        self.assertTrue(filename.startswith(os.getcwd()))
+        filename = filename[len(os.getcwd())+1:]
+        workspace = '%s/workspace'%self.test_dir
+        assertions = '%s/assertions.txt'%self.test_dir
+        file(assertions,'w').write('0x0000000000401003 ZF == 1')
```

Comment:
There is no `file` in Py3, this should be at least `open`, and the best would be `with open(assertions 'w') as f: f.write(...)`

Reply:
Since this is a test, it might be good to think of a named temporary file (see https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile)

Reply:
ok

Reply:
As for the temporary file, we are already inside a temporary directory, is not that enough ?

Reply:
> we are already inside a temporary directory, is not that enough ?

Yeah I think it is. The adventage of `NamedTemporaryFile` is that we don't have to remember to remove it at all as it has the `delete=True` default:
> If delete is true (the default), the file is deleted as soon as it is closed

Reply:
the temp directory is deleted after execution of one test with `shutil.rmtree(self.test_dir)`